shared/utils/find_untranslated.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ "$#" -lt 3 ] && {
+	echo "Usage:"
+	echo -e "\t$0 <original directory> <target directory> <path replacement ';' as separator>"
+
+	echo "Example:"
+	echo -e "\t$0 ./bash ./ansible '/(.*).sh;/\\1.yml'"
+	echo ""
+	echo "This file show 'diff' between directories & show not translated remediations"
```

Comment:
I suggest: This script compares two directories and looks for remediation scripts which are not implemented in one of the specified languages.